server/client.go

```diff
@@ -334,14 +334,31 @@ func MakeCreateIssueURL(instance Instance, project *jira.Project, issue *jira.Is
 
 	// add custom fields
 	for k, v := range issue.Fields.Unknowns {
-		strV, ok := v.(string)
-		if ok {
-			q.Add(k, strV)
-		}
-		if mapV, ok := v.(map[string]interface{}); ok {
-			if id, ok := mapV["id"].(string); ok {
+		switch vTyped := v.(type) {
```

Comment:
Are we handling all the cases here now? If not mention the same in the description with proper explanation.